lib/vagrant-libvirt/action/remove_stale_volume.rb

```diff
@@ -23,7 +23,9 @@ def call(env)
           fog_pool = ProviderLibvirt::Util::Collection.find_matching(
             env[:machine].provider.driver.connection.pools.all, config.storage_pool_name
           )
```

Comment:
Would it be worth just calling `@app.call(env)` if `fog_pool` is `nil` as presumably there is no need to run the remaining code.

Reply:
Yes, i thought about that too, the whole code doesnt need to be executed if ther eis acutally no image in the storage pool that is to be deleted.